apis/v1alpha2/policy_types.go

```diff
@@ -38,6 +38,15 @@ type PolicyTargetReference struct {
 	//
 	// +optional
 	Namespace *Namespace `json:"namespace,omitempty"`
+
+	// SectionName is the name of a section within the target resource. When
+	// unspecified, this targets the entire resource. In the following
```

Comment:
```suggestion
	// unspecified, this targetRef targets the entire resource. In the following
```
What "this" refers to is a little unclear, better to be specific.